api/v1/submissions/PKPSubmissionHandler.php

```diff
@@ -926,6 +926,7 @@ public function addPublication($slimRequest, $response, $args)
             return $response->withStatus(400)->withJson($errors);
         }
 
+        $params = (new \PKP\submission\Sanitizer)->sanitize($params, ['title']);
```

Comment:
I'd like at least some forms of invalid data to return error messages. For example, if I use `<strong>` but only `<b>` is in `allowed_html`, then I should get an error message that tells me what I've done wrong and how to fix it.

Maybe a single catch-all validation check is enough like this:

```
$sanitized = PKPString::stripUnsafeHtml($original);
if ($original !== $sanitized) {
    $errors['title'][$locale][] = 'Only the following HTML tags are allowed titles: <b>, <strong>, <em>, <i>, <sup>, <sub>.`
}
```